luvit-loader.lua

```diff
@@ -197,7 +197,7 @@ local function loader(dir, path, bundleOnly)
         return package.loaded[key]
       end
       local code = bundle.readfile(fullPath)
-      local module = loadstring(code, key)()
+      local module = loadstring(code, key)(key)
```

Comment:
Seems the '@' prepend should go on the second `loadstring` parameter, not passed to the loaded function. Something like:

```
local path = pathJoin(bundle.base, "./" .. fullPath)
local module = loadstring(code, path)(key)
```

?